src/android/CleverTapEventEmitter.java

```diff
@@ -0,0 +1,31 @@
+package com.clevertap.cordova;
+
+import android.util.Log;
+import org.apache.cordova.CordovaWebView;
+
+public class CleverTapEventEmitter {
+
+    private static final String LOG_TAG = "CleverTapEventEmitter";
+
+    private static CordovaWebView cordovaWebView;
```

Comment:
is it okay to cache this, what about setting this to null, any scenarios for memory leaks/other problems?

Reply:
Fixed in [8da2018](https://github.com/CleverTap/clevertap-cordova/pull/259/commits/8da2018d63ff570b9d9548d1b2d8cf5cd8c77747)